tests/Feature/crud/_02ReadTest.php

```
@@ -45,21 +45,28 @@ public function test_read_static_record(): void
+                    $this->assertEquals(AbstractMapper::PASSWORD_REPLACEMENT, $content->data[0]->$key);
+                    break;
+                default:
+                    $this->assertEquals($value, $content->data[0]->$key);
+            }
         }
     }
 
+    public function test_read_deleted_record(): void
+    {
+        $response = $this->get($this->endpoint.'test_table/999999');
+        $content = json_decode($response->getContent(), false);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$content'. (undefined)

(UnusedLocalVariable)